plugins/derive/g_derive.mlg

```diff
@@ -24,5 +24,5 @@ let classify_derive_command _ = Vernacextend.(VtStartProof (Doesn'tGuaranteeOpac
 
 VERNAC COMMAND EXTEND Derive CLASSIFIED BY { classify_derive_command }
 | ![ proof ] [ "Derive" ident(f) "SuchThat" constr(suchthat) "As" ident(lemma) ] ->
-  { fun ~pstate -> Some Derive.(start_deriving f suchthat lemma) }
+  { fun ~ontop:_ -> Some Derive.(start_deriving f suchthat lemma) }
```

Comment:
given that `ontop` is unused, can `![proof]` be omitted? Alternatively if `Derive.` cannot be nested, we should error when `ontop isSome`.

Reply:
> can `![proof]` be omitted?

Indeed that's a bug in derive, will fix as suggested thanks!

Reply:
Fixed.